doc/radosgw/ldap-auth.rst

```diff
@@ -14,13 +14,18 @@ search filter is constructed with the user name. The Ceph Object Gateway uses
 the configured service account to search the directory for a matching entry. If
 an entry is found, the Ceph Object Gateway attempts to bind to the found
 distinguished name with the password from the token. If the credentials are
-valid, the bind will succeed, and the Ceph Object Gateway will grant access.
+valid, the bind will succeed, and the Ceph Object Gateway will grant access and
+radosgw-user will be created with the provided username.
 
 You can limit the allowed users by setting the base for the search to a
 specific organizational unit or by specifying a custom search filter, for
 example requiring specific group membership, custom object classes, or
 attributes.
 
+The LDAP credentials must be available on the server to perform the LDAP
+authentication. Make sure to set the ``rgw`` log level low enough to hide the
+base-64-encoded credentials / access tokens.
```

Comment:
this sounds like a bug, not something to doc?

Reply:
well - thats by design. In order to authenticate agains an LDAP server, users need to send their password somehow to RGW, so RGW can do the auth against LDAP...
It's not clear "cleartext", but base64-encoded. I would still consider this cleartext, because everyone who looks at the RGW source code immediately knows how to extract the plain user password from RGW logs with level 20.

This is the same behaviour for normal RGW users, their AWS_ACCESS_KEY and AWS_SECRET_ACCESS_KEY are printed to the log with level 20 - right?

Reply:
(here a random example found on StackOverflow: https://stackoverflow.com/q/51986022, note that the string `ewogICAgIlJHV19UT0tFTiI6IH...` is printed many many times to the logs)

Reply:
I see what you mean, though possibly in this case, we need to handle the access key disjointly so as to suppress the byte sequence we received (e.g., we could substitute the DN);  what do you think?

Reply:
Yes, ideally we would refactor all RGW logging to *not* display sensitive information, either in general, or specific to the LDAP auth case.
However, this seems a bit out of scope for now - and doesn't solve this possible undocumented information leak in existing clusters.